docs/vNext/FISCO_BCOS_Configuration.md

```diff
@@ -0,0 +1,171 @@
+>*The required minimal version of FISCO BCOS is v2.0.0*
+
+FISCO BCOS is a secure and reliable financial-grade open-source blockchain platform led by Chinese enterprises. Its performance has reached over 10,000 TPS with single-chain setup. The platform provides rich features including group architecture, cross-chain communication protocols, pluggable consensus mechanisms, privacy protection algorithms, OSCCA-approved (Office of state Commercial Cryptography Administration) cryptography algorithms, and distributed storage. Its performance, usability and security has been testified by many institutional users and successful business applications in live production environment.
+
+### Adaptor Code
+
+The FISCO BCOS adaptor code can be found in the following folders:
+
+- `caliper/packages/caliper-samples/benchmark/fisco-bcos/`, contains three benchmark scripts and their configuration files::
+  - `helloworld`, a basic benchmark for testing getting/setting a string.
+  - `transfer/solidity`, a benchmark for testing transferring money between different accounts. Its smart contract is implemented in [solidity](https://solidity.readthedocs.io/en/latest/) programming language
+  - `transfer/precompiled`, a benchmark for testing transferring money between different accounts. Its smart contract is implemented by the [precompiled contract framework](https://fisco-bcos-documentation.readthedocs.io/en/latest/docs/design/virtual_machine/precompiled.html)  of FISCO BCOS.
+- `caliper/packages/caliper-samples/network/fisco-bcos/`, contains example configuration files for network configuration.
+- `caliper/packages/caliper-samples/src/contract/fisco-bcos/`, contains smart contracts used by benchmarks.
+- `caliper/packages/caliper-fisco-bcos/`, the code of FISCO BCOS adapter implementing the Caliper interfaces.
```

Comment:
Please remove - this is unrequired since the page should only be about configuring the adaptor